### PR TITLE
feat(star-warp): Implement jump to search for Starlight/Pagefind

### PR DESCRIPTION
**Pull request by Fryuni**

### CODE REVIEW

File: packages/star-warp/routes/client.astro [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import warpInfo from '@it-astro:star-warp:config';
+
+export const prerender = true;
+---
+
+<script is:inline define:vars={{ warpInfo }}>
+	const search = new URLSearchParams(window.location.search);
+	const query = search.get('q');
+
+	console.log({ warpInfo, search, query });
```
Comment (Adammatthiesen) [inline packages/star-warp/routes/client.astro:11]: is this still supposed to be here?  looks like something from testing...

File: docs/src/img/site-warp-chrome.png
Comment (Adammatthiesen) [inline docs/src/img/site-warp-chrome.png:1]: wait what.....? neat....

File: packages/star-warp/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+			env: 'dev',
+			trailingSlash: 'ignore',
+		};
+
+		const pathPattern = options.path.replace(/(^\/+|\/+$)/g, '');
+
+		const comboIntegration = {
+			name,
+			hooks: {
+				setup: (params: any) => {
```
Comment (Adammatthiesen) [inline packages/star-warp/src/index.ts:32]: oooooooooooooooh nice hook.... that is pretty neat.

**Review by Adammatthiesen — COMMENTED** [review]
Left 1 actual comment and 2 other just little notes thinking it is cool

Side note, i need to look into search stuff... :rofl:

File: packages/star-warp/routes/client.astro (hunk 1 above)
Comment (Fryuni) [reply packages/star-warp/routes/client.astro:11]: Good catch

**Pull request merged**